Rlisa33.md

```diff
@@ -0,0 +1,20 @@
+# **Enjoying Github Intern Software Delevopment Service!**
+[Rlisa33 Homepage](https://github.com/Rlisa33/Rlisa33.github.io)
+- (win_20170328_20_18_28_pro)
```

Comment:
Windows 10 Pro is sufficient -  the actual patch number will change all the time.